region/lowernorfair/east/Fast Pillars Setup Room.json

```diff
@@ -1054,6 +1139,7 @@
       "link": [5, 1],
       "name": "Jump into Respawning Block",
       "requires": [
+        {"noFlashSuit": {}},
```

Comment:
I couldn't do it at all between not having a high enough hitbox and activating the spark.  But I can try it again.

Reply:
Start from the right, against the closed door or a bit further if it's open. It shouldnt require anything different from the normal strat to avoid wasting the spark. Then once youre inside, an angle down + jump works fine.